.github/workflows/pull-target-e2e.yml

```diff
@@ -17,8 +17,19 @@ on:
       - "sec-scanners-config.yaml"
 
 jobs:
+  is-pr-approved:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if PR approved
+        env:
+          GH_TOKEN: ${{ github.token }}
+          PR_URL: https://github.com/kyma-project/eventing-manager/pull/${{ github.event.number }}
+        run: |
+          ./scripts/is_pr_approved.sh
+
   wait-until-build-succeeds:
     runs-on: ubuntu-latest
+    needs: is-pr-approved
```

Comment:
why don't you want to run this if an PR not approved?

Reply:
We have quota limits on number of gardener clusters, so we are filtering out to only run on approved PRs so less number of gardener clusters are created at a time.